clients/js/src/utilities.ts

```diff
@@ -129,3 +137,106 @@ export async function getOwnerFromAccount(
   }
   return accountInfo.value.owner;
 }
+
+export async function getMintFromTokenAccount(
+  rpc: Rpc<GetAccountInfoApi>,
+  tokenAccountAddress: Address,
+): Promise<Address> {
+  const account = await fetchEncodedAccount(rpc, tokenAccountAddress);
+  if (!account.exists) {
+    throw new Error(`Unwrapped token account ${tokenAccountAddress} not found.`);
+  }
+  return getTokenDecoder().decode(account.data).mint;
+}
+
```

Comment:
Everything below this comment was moved from `src/wrap.ts` given `combinedMultisigTx` is a shared function between both wrap and unwrap helpers.